conda-recipes/cyclus-deps/post-link.sh

```diff
@@ -1,30 +1,3 @@
-#/bin/bash
```

Comment:
First, I think conda explicitly calls `bash <script name>`. Second, this was missing the shebang anyway and wouldn't have worked =/